spec/integration_spec.rb

```diff
@@ -177,6 +177,15 @@
             expect(subject.license).to eql(license)
           end
         end
+
+        context 'with package.json license expression' do
+          let(:fixture) { 'npm-license-expression' }
+          let(:arguments) { { detect_packages: true } }
+
+          it 'matches other' do
+            expect(subject.license).to eql(other_license)
+          end
+        end
```

Comment:
Maybe this test should go in `npm_bower_matcher_spec.rb` instead, skipping the fixture? Is suspect that's the case, but I'll leave here for now pending review.

Reply:
👍